cmd/scollector/collectors/redis_linux.go

```diff
@@ -1,5 +1,3 @@
-// +build darwin linux
```

Comment:
Oh, collector build was broken at the time i was making this PR. Had to patch manually.